src/rgw/CMakeLists.txt

```diff
@@ -3,13 +3,19 @@ add_executable(ceph_rgw_jsonparser
   rgw_jsonparser.cc)
 target_link_libraries(ceph_rgw_jsonparser
   rgw_a
-  global rabbitmq)
+  global)
+if(WITH_RADOSGW_AMQP_ENDPOINT)
```

Comment:
sorry, i forgot to publish my comment.

@tserong i'd suggest link `RabbitMQ::RabbitMQ` instead of assuming the library name is rabbitmq. and we'd better check its existence before linking against this library. i prepared an alternative fix at #26667 . could you take a look at it?